lib/notifications/fcm/fcm_notifications_listener.dart

```diff
@@ -187,15 +155,13 @@ class FcmNotificationsListener {
   }
 
   /// Returns bool that indicates if push notifications are authorized
-  /// On Android it is always true
   Future<bool> isPushAuthorized() async {
     final notificationSettings = await _fcm.getNotificationSettings();
     return notificationSettings.authorizationStatus ==
         AuthorizationStatus.authorized;
   }
 
   /// Returns the current authorization status for push notifications
-  /// On Android it is always authorized
```

Comment:
This seemed like an informative comment unless it's false. The one in the method above too.